### DISCUSSION TITLE
why it isnt work

### DISCUSSION THREAD

**Discussion by ibrohim2911** [community/community#8838] [Copilot Conversations]
![Screenshot_2](https://user-images.githubusercontent.com/77937292/146043567-2d2f3beb-2f12-4da2-b252-57daed9ffb23.png)

**Comment by ibrohim2911** [comment]
there is deactivate but if i activate it, it isnt work

**Reply by davecheney** [reply]
Can you please past the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.
